### PR TITLE
processor_metrics_selector: Add documents for metrics_selector processor

### PR DESCRIPTION
**Pull request by cosmo0920**

### CODE REVIEW

File: installation/sources/build-and-install.md [hunk 1]
```diff
@@ -212,3 +212,10 @@ The _output plugins_ gives the capacity to flush the information to some externa
 | FLB\_OUT\_TCP | Enable TCP/TLS output plugin | On |
 | [FLB\_OUT\_TD](../../pipeline/outputs/treasure-data.md) | Enable [Treasure Data](http://www.treasuredata.com) output plugin | On |
 
+### Processor Plugins
+
+The _processor plugins_ gives the capability to handle the three types of events within the processor pipelines to allow modifying, enrich or drop events. The following table describes the processors available on this version:
```
Comment (patrick-stephens) [inline installation/sources/build-and-install.md:217]:
```suggestion
The _processor plugins_ provide the capability to handle the events within the processor pipelines to allow modifying, enrich or drop events. 
The following table describes the processors available on this version:
```

File: pipeline/processors/metrics-selector.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# Metrics Selector
+
+The **metric_selector** processor allows you to select metrics from the ingested metrics type of events.
```
Comment (patrick-stephens) [inline pipeline/processors/metrics-selector.md:3]:
```suggestion
The **metric_selector** processor allows you to select metrics to include or exclude (similar to the `grep` filter for logs).
```

File: pipeline/processors/metrics-selector.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+  outputs:
+    - name: stdout
+      match: '*'
+```
+{% endtab %}
+{% endtabs %}
+
+
+Note that processor is only effective on the YAML format. Also, processor elements should be located under input or output plguins' definitions.
```
Comment (patrick-stephens) [inline pipeline/processors/metrics-selector.md:55]: Probably move to the top I think.
```suggestion
All processors are only valid with the YAML configuration format. 
Processor configuration should be located under the relevant input or output plugin configuration.
```

File: pipeline/processors/metrics-selector.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+Metric\_Name parameter will translate the strings which is quoted with backslashes `/.../` as Regular expressions.
```
Comment (patrick-stephens) [inline pipeline/processors/metrics-selector.md:57]: Can we add an example? It's hard to understand

Comment (cosmo0920) [reply pipeline/processors/metrics-selector.md:58]: Already, they are in the example.... https://github.com/fluent/fluent-bit-docs/pull/1340/files#diff-f4f33fad5d0e68d91db11691404add8f8dc03ee9b53032d1ea4247f5e1bde02cR37

Comment (cosmo0920) [reply pipeline/processors/metrics-selector.md:58]: Yeah, I understand what you mean. So, I added a sentence for describe with an example string.

**Review by patrick-stephens — APPROVED** [review]

**Pull request merged**